guides/v2.1/install-gde/install/cli/install-cli-uninstall.md

```diff
@@ -70,23 +68,23 @@ The following message displays to confirm a successful uninstallation:
 
 	[SUCCESS]: Magento uninstallation complete.
 
-<h2 id="instgde-install-keep">Optionally keeping generated files</h2>
+## Optionally keeping generated files {#instgde-install-keep}
 By default, `magento setup:upgrade` clears compiled code and the cache. Typically, you use `magento setup:upgrade` to update components and each component can require different compiled classes.
 
 However, in some situations (particularly, deploying Magento to production), you might wish to avoid clearing compiled code because it can take some time. (The {% glossarytooltip 0bc9c8bc-de1a-4a06-9c99-a89a29c30645 %}cache{% endglossarytooltip %} is still cleared.) To update the Magento {% glossarytooltip 66b924b4-8097-4aea-93d9-05a81e6cc00c %}database schema{% endglossarytooltip %} and data *without* clearing compiled code, enter:
 
 	magento setup:upgrade --keep-generated
 
-<div class="bs-callout bs-callout-warning">
-    <p>The optional <code>--keep-generated</code> option should be used <em>only</em> in limited circumstances by experienced system integrators. <code>--keep-generated</code> should <em>never</em> be used in a development environment.</p>
-    <p>Improper use of this optional parameter can cause errors during code execution.</p>
-</div>
+{:.bs-callout .bs-callout-warning}
```

Comment:
This callout has multiple paragraphs, but no Liquid variables. This is the preferred format in such cases.

```
{%
include note.html
type='warning'
content='All of your markdown text, in multiple paragraphs.'
%}
```
(`info` and `tip` can also be used instead of `warning`. Make sure your content is enclosed with straight single quotes.)